packages/neutron-sdk/src/lib.rs

```diff
@@ -3,22 +3,18 @@
 // #![forbid(unsafe_code)]
 #![warn(trivial_casts, trivial_numeric_casts, unused_import_braces)]
 
+// TODO: can we get it from other place? or move NEUTRON_COMMIT to other place
+// TODO: do we need NEUTROND_VERSION?
```

Comment:
I don't think we need this constant anymore, since we are not building proto files here now